### ISSUE TITLE
Conversion rules end up in the failure message multiple times

### ISSUE THREAD

**Issue by dennisdoomen** [fluentassertions/fluentassertions#738]
```
With configuration:
- Use declared types and members
- Compare enums by value
- Match member by name (or throw)
- Try conversion of all members. Try conversion of member x.SelectedMemberPath.Contains("Birthdate"). Try conversion of all members. 
- Be strict about the order of items in byte arrays
" does not.
```

**Comment by dennisdoomen** [comment]
Can't reproduce

**Comment by dennisdoomen** [comment]
The `TryConversionStep` is added every time a new instance of the `SelfReferenceEquivalencyAssertionOptions` is created. First when it copies over the `UserEquivalencySteps` from the defaults (which always adds the `TryConversionStep` as part of it), and possibly later

**Comment by krajek** [comment]
I will tackle the problem. Any comments before I start? Is there any test that reproduces the behaviour? Is it only about duplication in the description or the behaviour itself is also affected?

**Comment by dennisdoomen** [comment]
It's purely a cosmetic issue.

**Comment by jnyrup** [comment]
@krajek I guess this can be closed with #941 ?
